cli/wallet/nep5.go

```diff
@@ -56,7 +56,7 @@ func newNEP5Commands() []cli.Command {
 			Name:  "amount",
 			Usage: "Amount of asset to send",
 		},
-		flags.Fixed8Flag{
+		cli.Int64Flag{
```

Comment:
That's not user-friendly. I think we need to keep the old notation here. For user-facing things there is a `BigDecimal` in C# node.